pages/public/index.html

```diff
@@ -110,8 +110,8 @@ <h1>Release 1.0.1 (Amadeus)</h1>
             It would mean a lot to us if you could report any problems you might have on GitHub or Discord linked below so we can further improve.
             Update JAN 7, 2024 - c1.0.1 released, find the changelog at the end of the release notes.
         </p>
-        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/download/release_1_0/deluge-community-release.zip">Download c1.0.1</a>
-        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/tag/release_1_0">Release notes</a>
+        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/latest/download/deluge-community-release.zip">Download c1.0.1</a>
+        <a class="btn" target="_blank" href="https://github.com/SynthstromAudible/DelugeFirmware/releases/latest">Release notes</a>
```

Comment:
we should use the exact tag instead, to avoid accidentally linking e.g. 1.1.0 when that release is created to host nightlies.

Reply:
Latest always points to the latest non-pre release build

Builds are adjusted by #937 to ensure that beta/nightly builds are marked as such